tests/CMakeLists.txt

```diff
@@ -10,8 +10,6 @@ function(ADD_PAGMO_TESTCASE arg1)
     add_test(${arg1} ${arg1})
 endfunction()
 
-file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/cec2013_data/" DESTINATION "${CMAKE_BINARY_DIR}/tests/cec2013_data/")
```

Comment:
I am a bit confused as why this stuff is here. It's because you branched before we merged cec2013 into master? Just hoping this does not come back from the graveyard :)